modules/custom/az_media/config/install/filter.format.az_standard.yml

```diff
@@ -28,7 +29,7 @@ filters:
     status: true
     weight: -50
     settings:
-      allowed_html: '<br> <p class> <h1 class name id> <h2 class name id> <h3 class name id> <h4 class name id> <h5 class name id> <h6 class name id> <a class target name id title href data-entity-type data-entity-uuid data-entity-substitution> <span class name id> <ul class type> <blockquote class> <footer class> <table class> <thead class> <abbr> <address> <cite> <dl> <dt> <dd> <div name id class data-*> <img src alt data-entity-type data-entity-uuid> <ol type class start> <li class> <tbody class> <tfoot class> <th class scope rowspan colspan> <td class rowspan colspan> <tr class colspan rowspan> <hr class> <drupal-entity data-entity-type data-embed-button alt title data-align data-caption data-entity-embed-display data-entity-embed-display-settings data-view-mode data-entity-uuid data-langcode> <drupal-media title data-entity-type data-entity-uuid alt data-view-mode data-align> <strong> <em> <code class="language-*"> <pre class="text-align-left text-align-center text-align-right text-align-justify"> <s> <sub> <sup> <caption>'
+      allowed_html: '<br> <p class> <h1 class name id> <h2 class name id> <h3 class name id> <h4 class name id> <h5 class name id> <h6 class name id> <a class target name id title href data-entity-type data-entity-uuid data-entity-substitution> <span class name id> <ul class type> <blockquote class> <footer class> <table class> <thead class> <abbr> <address> <cite> <dl> <dt> <dd> <div class name id data-*> <img src alt data-entity-type data-entity-uuid> <ol type class start> <li class> <tbody class> <tfoot class> <th class scope rowspan colspan> <td class rowspan colspan> <tr class colspan rowspan> <hr class> <drupal-entity data-entity-type data-embed-button alt title data-align data-caption data-entity-embed-display data-entity-embed-display-settings data-view-mode data-entity-uuid data-langcode> <drupal-media title data-entity-type data-entity-uuid alt data-view-mode data-align> <strong> <em> <code class="language-*"> <pre class="text-align-left text-align-center text-align-right text-align-justify"> <s> <sub> <sup> <caption>'
```

Comment:
Looks like the only change in this blob is attribute order.
<img width="1413" alt="image" src="https://github.com/az-digital/az_quickstart/assets/1023167/8af67f96-e5e6-4511-9215-271f9bbb9c93">